### PR TITLE
Add batch APIs

### LINKED ISSUE #73: Add batch APIs

### PR DESCRIPTION
**Pull request by Sagilio**
close #73

### CODE REVIEW

**Comment by hsluoyz** [comment]
@huazhikui @xcaptain

**Comment by hsluoyz** [comment]
@Sagilio please fix CI:

image

**Comment by Sagilio** [comment]
@hsluoyz  Already fixed.

**Review by huazhikui — APPROVED** [review]

File: NetCasbin/Model/Policy.cs [hunk 1]
```diff
@@ -91,29 +93,71 @@ public List<List<string>> GetFilteredPolicy(string sec, string ptype, int fieldI
 
         public bool HasPolicy(string sec, string ptype, List<string> rule)
         {
-            return Model[sec][ptype] != null && Model[sec][ptype].Contains(rule);
+            var assertion = GetExistAssertion(sec, ptype);
+            return assertion.Contains(rule);
+        }
+
+        public bool HasPolicies(string sec, string ptype, IEnumerable<List<string>> rules)
```
Comment (xcaptain) [inline NetCasbin/Model/Policy.cs:100]: This method is not test-coveraged, would be great to add some.

**Review by xcaptain — APPROVED** [review]

File: NetCasbin/Model/Policy.cs (hunk 1 above)
Comment (Sagilio) [reply NetCasbin/Model/Policy.cs:100]: Ok

Comment (Sagilio) [reply NetCasbin/Model/Policy.cs:100]: The `HasPolicies` method will be called at InternalAdd/RemovePolicies and the test has call those apis by e.Add/RemovePolicies.  The go casbin looks like also has not more test case.
I will add more tests later at the pull request for issue #76 .

**Pull request merged**